app/setup_handlers.go

```diff
@@ -18,7 +19,14 @@ func setUpgradeHandler(app *ElysApp) {
 	app.UpgradeKeeper.SetUpgradeHandler(version.Version, func(ctx sdk.Context, plan types.Plan, vm m.VersionMap) (m.VersionMap, error) {
 		app.Logger().Info("Running upgrade handler for " + version.Version)
 
-		return app.mm.RunMigrations(ctx, app.configurator, vm)
+		// Run migrations
+		verisionMap, err := app.mm.RunMigrations(ctx, app.configurator, vm)
+
+		// Set Params of incentive module with the new paramset added
+		params := incentivemoduletypes.DefaultParams()
+		app.IncentiveKeeper.SetParams(ctx, params)
```

Comment:
If previous upgrade handlers are done successfully, this process won't be required since RunMigrations is automatically running InitGenesis of not initialized modules and initGenesis would be setting the default params.
https://github.com/elys-network/elys/blob/feat/upgrade-incentive-params/x/incentive/keeper/genesis.go#L12

Probably, set params before RunMigrations?

Reply:
Yeah, I am trying to do chain upgrade in my local. Am not sure with that exactly, anyhow reproducing the error at the moment.

Reply:
@kenta-elys we should increament the incentive module version and add a migrator function to the module instead

Reply:
@cosmic-vagabond 
Could you commit this one? I am not sure how to do it.